utils/connectApp.js

```diff
@@ -74,6 +74,28 @@ const connectApp = async (req, res) => {
       return res.status(200).json({data: shaResult, code: 200});
     }
 
+    else if (api === 'getQuestSurveyFromGitHub') {
+      if (req.method !== 'GET') {
+        return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
+      }
+
+      if (!req.query.sha || req.query.sha === '') {
```

Comment:
This check is redundant; the empty string is already falsey.

Reply:
Good call. I cleaned it up. Thanks!